operations/agent-flow-mixin/dashboards/resources.libsonnet

```diff
@@ -42,6 +42,9 @@ local stackedPanelMixin = {
         label_values(agent_component_controller_running_components_total{cluster="$cluster", namespace="$namespace"}, instance)
       |||),
     ]) +
+    dashboard.withAnnotations([
+      dashboard.newLokiAnnotation('Deployments', '{cluster="$cluster", container="kube-diff-logger"} | json | namespace_extracted="grafana-agent" | name_extracted=~"grafana-agent.*"', 'rgba(0, 211, 255, 1)'),
```

Comment:
This will definitely be useful for us, but in the future when we recommend this mixin to everyone, we'll probably want to make this optional since not everyone will have it. Can we add in a TODO comment here so we can track making it optional?

Reply:
Yes, will do. Also, as I understand it, the annotation will have to be added on each dashbord separately, is that correct?

Reply:
As it is now, yeah, but you could update `dashboard.new` so it always adds the annotation if you wanted it to be available by default.